views/partials/banners/download_firefox_bar.hbs

```diff
@@ -0,0 +1,8 @@
+<div id="download-firefox-bar" class="download-firefox download-firefox-bar banner">
+  <div class="section-wrapper">
+    <div class="whole">
+      <p>Firefox Monitor is brought to you by the all-new Firefox. <a class="internal-link" href="https://www.mozilla.org/firefox/new/?utm_source={{ UTM_SOURCE }}&utm_medium=referral&utm_campaign=fx_monitor_downloads&utm_content=download_banner" target="_blank" rel="noopener">Download Firefox now &#187;</a>
```

Comment:
We should make this `utm_content` value different than the one lower on the page. So we can see how effective each link is on its own. Suggest `utm_content=top_download_banner`